test/integration/test_quota.py

```diff
@@ -85,6 +85,27 @@ def test_update(self):
         json_response = show_response.json()
         assert json_response["name"] == new_quota_name
 
+    def test_update_description(self):
+        quota_name = "test-update-quota-description"
+        quota = self._create_quota_with_name(quota_name)
+        quota_id = quota["id"]
+
+        # update description (one needs to specify a name even if should not be changed)
+        quota_description = "description of test-updated-quota-name"
+        update_payload = {
+            "name": quota_name,
+            "description": quota_description,
+        }
+        put_response = self._put(f"quotas/{quota_id}", data=update_payload, json=True)
+        put_response.raise_for_status()
+        assert "has been renamed to" in put_response.text
```

Comment:
It's a good idea not to assert messages that are pretty questionable, considering  that having to set the name doesn't make sense and that we're not even renaming anything. 
```suggestion
```